### ISSUE TITLE
[Feature Request]: Official Docker images on ARM64

### ISSUE THREAD

**Issue by venom85** [grokability/snipe-it#12099]
### Is your feature request related to a problem? Please describe.

I would like to run this on a Kubernetes cluster of ARM64 Raspberry Pis. Right now, the only official image is x86_64, so I get an invalid architecture error.

### Describe the solution you'd like

There are alternative images that support ARM64, but none of them will come with the support from this group that an official image would. It would be great if ARM64 images could be officially added to the Docker Hub listing.

### Describe alternatives you've considered

There are other, unofficial images, like the one from linuxserver.

**Comment by NojoudAlshehri** [comment]
Issue Already Fixed. Original issue is #8342.
Solved in Pull request #12904 => Fixed #8342: Add ARM64 architecture to Docker Builds

**Comment by uberbrady** [comment]
Yes, as soon as we stamp out a 6.1.1 release you _should_ find an ARM build there.

**Comment by ElDavoo** [comment]
This issue should be updated as arm64 docker images are available, however they are only available **sometimes**, not every image gets pushed with arm64 support.  I guess someone still did not update their buid-and-upload script and someone else did. Or maybe is done manually and snipe or whoever forgets the arm build.
For example ATTW latest does not have arm, but latest-alpine does.
![immagine](https://github.com/snipe/snipe-it/assets/7345120/b1d84ea9-350e-403b-9e97-669586f09966)

**Comment by uberbrady** [comment]
Yeah, we had a few build problems in there with keys expiring or whatever - next tagged release should hopefully work at least a little bit better.
